ethpm/deployments.py

```diff
@@ -32,7 +32,8 @@ def __contains__(self, key: str) -> bool:
 
     def get(self, key: str) -> Dict[str, str]:
         self._validate_name_and_references(key)
-        return self.deployment_data.get(key)
+        # type ignored b/c _validate_name_and_references enforces key exists
+        return self.deployment_data.get(key)  # type: ignore
```

Comment:
With the comment above why not use `return self.deployment_data[key]` since we know the key exists?